index.html

```diff
@@ -318,5 +318,61 @@ <h3 class="about-titles" id="sk-title">Skills</h3>
         </ul>
       </div>
     </section>
+
+    <footer class="contact-form headline">
+      <form
+        id="fs-frm"
+        name="simple-contact-form"
+        accept-charset="utf-8"
+        action="https://formspree.io/f/xwkyqzan"
+        method="POST"
+        class="intro conctact-me-form"
+      >
+        <fieldset id="conctact-me-inputs">
+          <div class="title-para-contact">
+            <label class="name-contact-title" for="full-name">Contact me</label>
+            <p id="para-contact-me">
+              If you have an application you are interested in developing, a
+              feature that you need built or a project that needs coding. I’d
+              love to help with it
+            </p>
+          </div>
+
+          <input
+            type="text"
+            name="name"
+            id="full-name"
+            placeholder="   Sebastian Solis Fenger"
+            required=""
+            maxlength="30"
+          />
+          <input
+            type="email"
+            name="_replyto"
+            id="email-address"
+            placeholder="   solisfenger77@gmail.com"
+            required=""
+          />
+          <textarea
+            rows="5"
+            name="message"
+            id="message"
+            placeholder="  Write your message here"
+            required=""
+            maxlength="500"
+          ></textarea>
```

Comment:
[ Required Changes ]
- [ ]  It would be nice if you stick to the Figma design as much as possible and adjust the styles of your input fields to match the real design.
- [ ] Please adjust the padding of your input fields in your CSS styles to match the real design
- Try these styles: 
`form input, 
textarea {
  padding: 15px 16px;
}`
- [ ] Kindly consider removing the border on the button and implement 





- Your Design
![image](https://user-images.githubusercontent.com/50941074/159545311-56f66430-e34d-4893-8065-36fbd73bf411.png)

- The Figma Design
![image](https://user-images.githubusercontent.com/50941074/159545698-42c3788b-011b-4bd7-b55b-50bf5008b170.png)

Reply:
All good now 👍🏽